CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Versions
 
+## x.x.x
+    * Depend on Android SDK 13.1.0 and iOS SDK 13.1.0.
```

Comment:
Let's keep the changelog for SDK version bumps consistent?

https://github.com/AppLovin/AppLovin-MAX-Cordova/commit/b33ed1eba91b33a9567a41eaa2934e00281b363c#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR4